docs/core/testing/unit-testing-with-dotnet-test.md

```diff
@@ -144,21 +112,50 @@ at the root of `project.json`:
 }
 ```
 
-Finally, you need to set the framework node to use
+The template also sets the framework node to use
 `netcoreapp1.0`, and include the required imports to
-get xUnit.net to work with RC2:
+get xUnit.net to work with .NET Core RTM:
 
 ```json
   "frameworks": {
     "netcoreapp1.0": {
       "imports": [
-        "dnxcore50",
+        "dnxcore54",
```

Comment:
It should be dotnet54.  Chatted with @blackdwarf this morning, and to go with consistency, I'm matching what's generated by the `dotnet new` command.  

e.g. this for console apps: https://github.com/dotnet/cli/blob/rel/1.0.0-preview2/src/dotnet/commands/dotnet-new/CSharp_Console/project.json.template

Does that make sense?